src/states/elder/mod.rs

```diff
@@ -1596,16 +1628,22 @@ impl Approved for Elder {
         payload: OnlinePayload,
         outbox: &mut dyn EventBox,
     ) -> Result<(), RoutingError> {
+        let _ = self
```

Comment:
Not sure about that.

Reply:
I think there is a bit of an issue in that we could get the Poke event before reaching that point. We probably should have taken it into account and not ignored it.

Reply:
This is currently OK because the `MemberKnowledge` messages are sent periodically. But If we want to change that then this can become an issue.

Reply:
I think it could well go all wrong:
 - Node is relocated to our section.
 - Node is Approved -> Approval is sent
 - Node send MemberKnowledge to all elders
 - Most have not yet approved it and ignore it as a result.
 - Node is promoted to Elder
 - now looking for Parsec last always, all further MemberKnowledge are ignored.
 - Node never receive the gossip from the version it expects and never see itself promoted.

Reply:
Yeah, this could be an issue. Perhaps we need to keep the parsec poke thing periodic.

Reply:
This is a problem even if parsec Poke is periodic, it is related to the use of MemberKnowledge to reply to a parsec poke.

Reply:
No, with periodic parsec poke (or `MemberKnowledge`), the elders do eventually get updated and will send the correct version gossip to the promoted node. Or am I missing something?

Reply:
I'm thinking if we consider a node an elder we ignore MemberKnowledge, so if that node is  lagging at every poke we keep on sending them the new parsec version while they need the old one to catch up.